doc/themes/the-things-stack/layouts/shortcodes/tabs/container.html

```diff
@@ -0,0 +1,77 @@
+<div class="tabs" id="tabs">
+  <ul>
+    <span> Show instructions for:</span>
+    <li class="is-active" data-tab="console">
+      <a>
+        <span>Console</span>
+      </a>
+    </li>
+    <li data-tab="cli">
+      <a>
+        <span>CLI</span>
+      </a>
+    </li>
```

Comment:
if we just have console instructions, i don't think we don't need to use tabs. i hardcoded the cli and console shortcodes to avoid things like spelling errors and to limit it to this use case with error catching, but i'll refactor so we can expand it to other use cases.